integration-tests/README.md

```diff
@@ -2,35 +2,43 @@
 
 Here lives the integration tests for chainlink, utilizing our [chainlink-testing-framework](https://github.com/smartcontractkit/chainlink-testing-framework).
 
-## How to Configure
+## Setup
 
-See the [example.env](./example.env) file to set some common environment variables to configure how tests are run.
+Prerequisites to run the tests.
 
-### Connect to a Kubernetes Cluster
+### Install Go
 
-Integration tests require a connection to an actively running kubernetes cluster. [Minikube](https://minikube.sigs.k8s.io/docs/start/)
-can work fine for some tests, but in order to run more rigorous tests, or to run with any parallelism, you'll need to either
-increase minikube's resources significantly, or get a more substantial cluster.
-This is necessary to deploy ephemeral testing environments, which include external adapters, chainlink nodes and their DBs,
-as well as some simulated blockchains, all depending on the types of tests and networks being used.
+[Install](https://go.dev/doc/install)
 
 ### Install Ginkgo
 
 [Ginkgo](https://onsi.github.io/ginkgo/) is the testing framework we use to compile and run our tests. It comes with a lot of handy testing setups and goodies on top of the standard Go testing packages.
 
 `go install github.com/onsi/ginkgo/v2/ginkgo`
 
+### Install NodeJS
+
+[Install](https://nodejs.org/en/download/)
```

Comment:
Do we want to simplify all of these install's to using asdf since all we would need to do is add helm to the .tool-versions and then here we would just need to say to install asdf and run `asdf install` to get helm, golang, and nodejs. We had this in the chainlink-testing-framework but now that the tests are over here we should move all that stuff here as well and make a proper `make install_qa_tools` or something like that that will do all of the setup for you in one go. Just a thought.

Reply:
I'm up for it. I'd nominate you to write that up though, as I don't think I've ever used `asdf` before.

Reply:
Can do